roles/openshift_health_checker/openshift_checks/disk_availability.py

```diff
@@ -27,10 +27,12 @@ def is_active(cls, task_vars):
     def run(self, tmp, task_vars):
         group_names = get_var(task_vars, "group_names")
         ansible_mounts = get_var(task_vars, "ansible_mounts")
-
-        min_free_bytes = max(self.recommended_disk_space_bytes.get(name, 0) for name in group_names)
         free_bytes = self.openshift_available_disk(ansible_mounts)
 
+        recommended_min = max(self.recommended_disk_space_bytes.get(name, 0) for name in group_names)
+        configured_min = int(get_var(task_vars, "openshift_check_min_host_disk_gb", default=0)) * 10**9
```

Comment:
@sosiouxme this variable seems to be too generic, how was this supposed to work?

I'm updating this check now to check multiple paths / mount points (/var, /usr/local/bin, /tmp or equivalent), and in that case this variable becomes ambiguous, which filesystem should I apply it to?

Is it used somewhere we know of?